styles/designs/cardboard/parts/_exercises-components.scss

```diff
@@ -16,6 +16,14 @@ $InjectedExercise__Container: (
     )
 );
 
+$ExerciseMultipartQuestion__Container: (
+    _name: "ExerciseMultipartQuestionContainer",
+    _subselector: '',
+    _properties: (
+        display: enum('ValueSet:::OPTIONAL'), // used in inline exercises
```

Comment:
There are no inline exercises in Cardboard, but I left it as it is in Carnival. Maybe we will need this in the future.